projects/plugins/wpcomsh/feature-plugins/full-site-editing.php

```diff
@@ -110,6 +113,7 @@ function wpcom_public_coming_soon_replace_yoast_seo_notice() {
 	// Only show the notice on certain admin pages and Yoast SEO admin pages.
 	if (
 		$should_replace_yoast_notice
+		&& isset( $wpseo_options['ignore_search_engines_discouraged_notice'] )
 		&& $wpseo_options['ignore_search_engines_discouraged_notice'] == true // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
```

Comment:
`! empty()` here too, which would also resolve the Universal.Operators.StrictComparisons.LooseEqual sniff.